src/input_cp2k_almo.F

```diff
@@ -888,21 +881,21 @@ SUBROUTINE create_penalty_section(section)
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
-      CALL keyword_create(keyword, __LOCATION__, name="OCCUPIED_VOLUME_PENALTY_COEFF", &
+      CALL keyword_create(keyword, __LOCATION__, name="PENALTY_STRENGTH", &
          description="Adjust the decrease speed of volume penalty coefficient cp in each step.", &
-         usage="VOLUME_PENALTY_COEFF_DECREASE_FACTOR 1.1", default_r_val=1.1_dp)
+         usage="PENALTY_STRENGTH 1.1", default_r_val=1.1_dp)
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
-      CALL keyword_create(keyword, __LOCATION__, name="VOLUME_PENALTY_COEFF_DECREASE_FACTOR", &
+      CALL keyword_create(keyword, __LOCATION__, name="PENALTY_STRENGTH_DECREASE_FACTOR", &
          description="Adjust the decrease speed of volume penalty coefficient cp in each step.", &
-         usage="VOLUME_PENALTY_COEFF_DECREASE_FACTOR 1.1", default_r_val=1.1_dp)
+         usage="PENALTY_STRENGTH_DECREASE_FACTOR 1.1", default_r_val=1.1_dp)
       CALL section_add_keyword(section, keyword)
       CALL keyword_release(keyword)
 
-      CALL keyword_create(keyword, __LOCATION__, name="DETERMINANT_ERROR", &
+      CALL keyword_create(keyword, __LOCATION__, name="DETERMINANT_TOLERANCE", &
```

Comment:
description = "Stop the optimization of the penalty strength if the determinant of the overlap changes less than this tolerance threshold."